### DISCUSSION TITLE
quantmod dependency issue: maximum of

### DISCUSSION THREAD

**Discussion by dsenkyr** [explosion/spaCy#11309] [Help: Other Questions] [answered]
## How to reproduce the behaviour
```
doc = nlp("Each project should be supervised by a maximum of 2 teachers.")

token_a = doc[6]
print(f"{token_a} <-{token_a.dep_}- {token_a.head}")

token_maximum = doc[7]
print(f"{token_maximum} <-{token_maximum.dep_}- {token_maximum.head}")

token_of = doc[8]
print(f"{token_of} <-{token_of.dep_}- {token_of.head}")
```

### Current output
```
a <-quantmod- 2
maximum <-quantmod- 2
of <-quantmod- 2
```

### Expected output
Only the `maximum` token has the `quantmod` dependency.

## Your Environment
- **spaCy version:** 3.4.1
- **Platform:** Windows-10-10.0.19044-SP0
- **Python version:** 3.10.6
- **Pipelines:** en_core_web_trf (3.4.0)

**Comment by polm** [comment]
Thanks for the report, that does look kind of weird! We will check to see if this is not a wider issue. For the record, with the 3.3.0 trf or the 3.4.0 large pipeline this is the output:

```
a <-det- maximum
maximum <-pobj- by
of <-prep- maximum
```

**Comment by adrianeboyd** [comment, marked as answer]
I checked and the training data is just inconsistent here. I'm not sure whether it's from original OntoNotes annotation or related to the dependency conversion, but that's why's it's inconsistent.
